packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -305,6 +311,11 @@ class RefreshIndicator extends StatefulWidget {
   /// Defaults to [RefreshIndicatorTriggerMode.onEdge].
   final RefreshIndicatorTriggerMode triggerMode;
 
+  /// Defines `elevation` for [RefreshIndicator].
+  ///
+  /// Defaults to 2.0.
```

Comment:
```suggestion
  /// Defines the elevation for the underlying [RefreshIndicator].
  ///
  /// Defaults to 2.0.
```
Super nit

Reply:
Perhaps even an `/// Defines the elevation of the underlying [RefreshIndicator].`?